src/main/java/com/bullhornsdk/data/model/entity/core/paybill/rate/PlacementRateCardLine.java

```diff
@@ -57,179 +66,8 @@ public class PlacementRateCardLine extends CustomFieldsI implements QueryEntity,
-        return placementRateCardLineGroup;
-    }
-
-    @JsonProperty("placementRateCardLineGroup")
-    public void setPlacementRateCardLineGroup(PlacementRateCardLineGroup placementRateCardLineGroup) {
-        this.placementRateCardLineGroup = placementRateCardLineGroup;
-    }
-
-    public BigDecimal getPayMultiplier() {
-        return payMultiplier;
-    }
-
-    public void setPayMultiplier(BigDecimal payMultiplier) {
-        this.payMultiplier = payMultiplier;
-    }
-
-    public BigDecimal getBillMultiplier() {
-        return billMultiplier;
-    }
-
-    public void setBillMultiplier(BigDecimal billMultiplier) {
-        this.billMultiplier = billMultiplier;
-    }
-
-    public BigDecimal getMarkupPercent() {
-        return markupPercent;
-    }
-
-    public void setMarkupPercent(BigDecimal markupPercent) {
-        this.markupPercent = markupPercent;
-    }
-
-    public BigDecimal getMarkupValue() {
-        return markupValue;
-    }
-
-    public void setMarkupValue(BigDecimal markupValue) {
-        this.markupValue = markupValue;
-    }
-
-    @Override
-    public DateTime getDateLastModified() {
-        return dateLastModified;
-    }
-
-    public void setDateLastModified(DateTime dateLastModified) {
-        this.dateLastModified = dateLastModified;
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-        if (!super.equals(o)) return false;
-        PlacementRateCardLine that = (PlacementRateCardLine) o;
-        return Objects.equals(id, that.id) && Objects.equals(alias, that.alias) && Objects.equals(billCurrencyUnit, that.billCurrencyUnit) && Objects.equals(billRate, that.billRate) && Objects.equals(earnCode, that.earnCode) && Objects.equals(externalID, that.externalID) && Objects.equals(migrateGUID, that.migrateGUID) && Objects.equals(payCurrencyUnit, that.payCurrencyUnit) && Objects.equals(payRate, that.payRate) && Objects.equals(placementRateCardLineGroup, that.placementRateCardLineGroup);
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(super.hashCode(), id, alias, billCurrencyUnit, billRate, earnCode, externalID, migrateGUID, payCurrencyUnit, payRate, placementRateCardLineGroup);
-    }
-
-    @Override
-    public String toString() {
-        return "PlacementRateCardLine{" +
-            "id=" + id +
-            ", alias='" + alias + '\'' +
-            ", billCurrencyUnit=" + billCurrencyUnit +
-            ", billRate=" + billRate +
-            ", earnCode=" + earnCode +
-            ", externalID='" + externalID + '\'' +
-            ", migrateGUID='" + migrateGUID + '\'' +
-            ", payCurrencyUnit=" + payCurrencyUnit +
-            ", payRate=" + payRate +
-            ", placementRateCardLineGroup=" + placementRateCardLineGroup +
-            '}';
-    }
-
+    private BigDecimal taxableMargin;
```

Comment:
Do you want to add a Getter/Setter for taxableMargin?

Reply:
It's added automatically by the Data annotation

Reply:
Nvm. There's a `@Data` annotation